scripts/ft-benchmark-data-sender.py

```diff
@@ -86,7 +86,7 @@ def get_commit() -> tuple[str, datetime]:
 
 def commit_to_db(data: dict) -> None:
     print(data)
-    chdir(os.path.expanduser("~/nearcore/benchmarks/continous/db/tool"))
+    chdir(os.path.expanduser("~/nearcore/benchmarks/continuous/db/tool"))
```

Comment:
cc @mooori @nagisa FYI in case any other tooling depends on this typo.

Reply:
The directory `benchmarks/continous` should be renamed accordingly. I can send a follow up PR for that.

Reply:
@InventiveCoder This PR merge failed anyway for unrelated reasons, can you either do the suggestion above or revert the relevant changes?

Reply:
To do the change here it is sufficient to:

```
git mv benchmarks/continous benchmarks/continuous
```

Reply:
I see there is already a PR to modify the directory name. Do I need to roll back the changes here?

Reply:
Reverted to be able to merge without conflicts